tempesta_fw/sched/tfw_sched_rr.c

```diff
@@ -33,20 +34,103 @@ MODULE_LICENSE("GPL");
 #define ERR(...) TFW_ERR(BANNER __VA_ARGS__)
```

Comment:
You almost fully rewrote the scheduler code, please update minor version on such large changes. If the code gets large new functionality or redesign, then minor version is updated. If it gets significant bugfixes, then patch level version is updated.